entrypoint.py

```diff
@@ -72,6 +72,8 @@ def unregister(cls, entrypoint_class):
         cls.ENTRY_POINTS.remove(entrypoint_class)
         del cls.BY_INTERNAL_NAME[entrypoint_class.INTERNAL_NAME]
         del cls.DISPLAY_TITLES[entrypoint_class]
+        if entrypoint_class in cls.DEFAULT_ENABLED:
+            cls.DEFAULT_ENABLED.remove(entrypoint_class)
```

Comment:
A mock EntryPoint was sticking around DEFAULT_ENABLED after the test was over and causing trouble.